src/adv.py

```diff
@@ -1,29 +1,35 @@
 from room import Room
+from player import Player
+from item import Item
+from helper_functions import handle_direction, get_item, handle_action
 
 # Declare all the rooms
 
+# This is a dictionary where the key is a the name of a room, and the value is the invocation of a Room class, with two strings passed in a parameters.
 room = {
     'outside':  Room("Outside Cave Entrance",
-                     "North of you, the cave mount beckons"),
+                     "North of you, the cave mount beckons", [Item("pebble"), Item("stick"), Item("doughnut")]),
```

Comment:
Nice idea adding it to the room definitions when I did it I used something similar to the way they instantiated the rooms.